server/src/test/scala/cromwell/engine/workflow/WorkflowActorSpec.scala

```diff
@@ -187,6 +187,27 @@ class WorkflowActorSpec extends CromwellTestKitWordSpec with WorkflowDescriptorB
       deathwatch.expectTerminated(actor)
     }
 
+    "abort execution before escalating failure if one of it's child actors crashed" in {
+      val actor = createWorkflowActor(ExecutingWorkflowState)
```

Comment:
A more descriptive name like `workflowActor` might make it easier to follow the code further on